### DISCUSSION TITLE
Any way to get rid of the « and » in pagination buttons?

### DISCUSSION THREAD

**Discussion by twrschm** [facebook/docusaurus#5385] [Q&A] [answered]
Hi there, I'm currently building a documentation page using Docusaurus 2.0.0 and now I got stuck when trying to visually change the "next" and "previous" buttons located at the bottom of every docs article.
In detail, I want to change (or at least get rid) of these arrows (« ») which are located down there (see screenshot / link below).

Is there any way to achieve this? Thanks for your help :)

[https://ibb.co/0F45dC7](https://ibb.co/0F45dC7)

**Comment by Josh-Cena** [comment, marked as answer]
Run `yarn swizzle @docusaurus/theme-classic DocPaginator` and get rid of the `&laquo;` & `&raquo;`

More about swizzling: https://docusaurus.io/docs/next/using-themes#swizzling-theme-components

**Reply by twrschm** [reply]
Hi @Josh-Cena, thanks for your quick reply last week!

I managed to swizzle the component (which only worked for me using the `--danger` flag). After editing the required file, nothing noticeable changes; the `&laquo;` and `&raquo;` characters are still there.
Is there any other step necessary in order to make Docusaurus use the modified component? Swizzle docs say that this should happen automatically...

Thank You!🙂

**Reply by Josh-Cena** [reply]
Yes, that should be the case... Did you restart your dev server?

Also, it could be that you are on a buggy version, so make sure you are using `beta.4`

**Reply by twrschm** [reply]
Yea I did. I'm currently on beta-3, I will update my version and try again. Thanks so far 👍

**Reply by slorber** [reply]
I think we have a webpack caching bug in beta3, so using beta.4 or `docusaurus clear` should fix it

**Reply by twrschm** [reply]
@Josh-Cena @slorber Though I had some issues while updating to beta-4 which took some time, it all works now and finally did the trick regarding the swizzled component. I ran the clear command for some clean-up as well.
Thanks a lot for your help! :)
